content/overview/pricing/rates.md

```diff
@@ -78,6 +78,16 @@ Free of charge and without any paperwork, anyone with a government email address
 
 [Learn more about sandboxes.]({{< relref "overview/pricing/free-limited-sandbox.md" >}})
 
+## Quota pricing
+
+Quotas are priced at $0.0033 per MB reserved per day and assessed monthly. In practice, this works out to $99 per GB per month, based on a 30-day month:
+
+`$0.0033 * 1000 MB * 30 days = $99 per GB per month`
+
+The monthly cost will vary slightly from $92.40 per GB in a 28-day month to $102.30 per GB in a 31-day month.
```

Comment:
Yes, we should have this detail available so that everyone can make accurate estimates and check their bills. $10 isn't meaningful for a budget, but it can be meaningful for making everyone's bookkeeping line up precisely, and that can be important as well.